monitoring/iscsi_test.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package monitoring
+
+import (
+	"fmt"
+
+	"github.com/gravitational/satellite/agent/health"
+	pb "github.com/gravitational/satellite/agent/proto/agentpb"
+	"github.com/gravitational/satellite/lib/test"
+
+	"github.com/coreos/go-systemd/v22/dbus"
+	. "gopkg.in/check.v1"
+)
+
+const (
+	ActiveAndMasked   = "ActiveState is inactive and LoadState is masked."
```

Comment:
nit: these need not be exported.
Also, `ActiveAndMasked` is actually `InactiveAndMasked`